cores/esp8266/core_esp8266_postmortem.cpp

```diff
@@ -174,17 +199,32 @@ void __wrap_system_restart_local() {
 
     // amount of stack taken by interrupt or exception handler
     // and everything up to __wrap_system_restart_local
-    // (determined empirically, might break)
+    // ~(determined empirically, might break)~
     uint32_t offset = 0;
     if (rst_info.reason == REASON_SOFT_WDT_RST) {
-        offset = 0x1a0;
+        // Stack Tally
+        // 256 User Exception vector handler reserves stack space
+        //     directed to _xtos_l1int_handler function in Boot ROM
+        //  48 wDev_ProcessFiq - its address appears in a vector table at 0x3FFFC27C
+        //  16 ?unnamed? - index into a table, pull out pointer, and call if non-zero
+        //     appears near near wDev_ProcessFiq
+        //  32 pp_soft_wdt_feed_local - gather the specifics and call __wrap_system_restart_local
+        offset =  32 + 16 + 48 + 256;
     }
     else if (rst_info.reason == REASON_EXCEPTION_RST) {
-        offset = 0x190;
+        // Stack Tally
+        // 256 Exception vector reserves stack space
+        //     filled in by "C" wrapper handler
+        //  16 Handler level 1 - enable icache
+        //  64 Handler level 2 - exception report
+        offset = 64 + 16 + 256;
     }
     else if (rst_info.reason == REASON_WDT_RST) {
         offset = 0x10;
     }
+    else if (rst_info.reason == REASON_USER_SWEXCEPTION_RST) {
```

Comment:
Also update ^ `REASON_WDT_RST` to decimal?

Reply:
done